### PR TITLE
Bug1711781 5.7

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
http://jenkins.percona.com/job/mysql-5.7-param/1206/

### CODE REVIEW

File: mysql-test/extra/binlog_tests/enforce_gtid_consistency_tmp_violation.test [hunk 1]
```diff
@@ -82,5 +84,6 @@ CREATE TEMPORARY TABLE IF NOT EXISTS t1 (a INT) ENGINE = MyISAM;
 SET AUTOCOMMIT = 0;
 --let $statement= DROP TEMPORARY TABLE t1
 --source extra/binlog_tests/enforce_gtid_consistency_statement.inc
+--let $sync_point= before_execute_sql_command
```
Comment (percona-ysorokin) [inline mysql-test/extra/binlog_tests/enforce_gtid_consistency_tmp_violation.test:87]: Should this line be before `--source extra/binlog_tests/enforce_gtid_consistency_statement.inc`?

Comment (laurynas-biveinis) [reply mysql-test/extra/binlog_tests/enforce_gtid_consistency_tmp_violation.test:87]: before_execute_sql_command is the default upstream sync point here. The idea here was to restore it immediately after the part where this patch necessitates something else, both logically and for a smaller diff

Comment (percona-ysorokin) [reply mysql-test/extra/binlog_tests/enforce_gtid_consistency_tmp_violation.test:87]: Oh, OK then.

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Review by robgolebiowski — APPROVED** [review]
LGTM

**Pull request merged**